libs/wingc/src/jsify/snapshots/base_class_captures_preflight.snap

```diff
@@ -118,8 +121,11 @@ class $Root extends $stdlib.std.Resource {
           })())
         `;
       }
-      _supportedOps() {
```

Comment:
Just to make sure I understand, so now onLiftDeps is used to determine supportedOps ?

Reply:
Yep. The idea I had was to replace `supportedOps` with `_liftDeps` since they're provide the same information